### DISCUSSION TITLE
Post your Flix Package!

### DISCUSSION THREAD

**Discussion by magnus-madsen** [flix/flix#5683] [Show and tell]
A place to discuss and promote your Flix packages.

**Comment by stephentetley** [comment]
I've finally got Flix Parsec working again and made a v0.1.0 release. It had been broken since last year mostly due to not playing well with effects but it now works well with the graded versions of Functor, Applicative and Monad. 

https://github.com/stephentetley/flix-parsec

I'd say it's still a bit too rough to depend on but it is a nice example of a "2 CPS" monad (i.e. with both success and failure continuations) which has been a goto method for implementing parser combinators in strict languages since Chris Okasaki's "Even Higher Order Functions for Parsing".

**Comment by LionelMeli** [comment]
Testing some ideas here: https://github.com/LionelMeli/Flix_Stdlib_Range  from issues like:
- [foreach questions #6304](https://github.com/flix/flix/discussions/6304)
- [Add Range module #7128](https://github.com/flix/flix/issues/7128)
- [hard edges of foreach #7178](https://github.com/flix/flix/issues/7178)
- [Reverse iteration of collections #6998](https://github.com/flix/flix/issues/6998)

**Comment by JonathanStarup** [comment]
https://github.com/JonathanStarup/ListSet for when you want a Set buy only have `Eq[t]`!
